plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -188,7 +188,7 @@ export default class NodesTable extends React.Component<
     const { data, sortColumn, sortDirection } = this.state;
 
     return (
-      <div style={{ flexGrow: 1 }}>
+      <div className="table-wrapper" style={{ flexGrow: 1 }}>
```

Comment:
Perhaps put the inline-styling as part of the `table-wrapper` class?

Reply:
Agree.

Reply:
What do you mean by not accessible. It works when tried.

Reply:
nvm, I forgot it's `grey-dark` instead of `greyDark`